AllJoyn/Platform/Bridge.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.24720.0
+# Visual Studio 15
```

Comment:
#Resolved

---
In reply to: [75179512](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75179512) [](ancestors = 75179512,75178273,75173869)